### PR TITLE
Hello-Microverse Project pull Request

### PR DESCRIPTION
**Pull request by kkasaija**
A basic HTML and CSS to demonstrate the use and application of linters

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+test.node_modules
```
Comment (mahbubul14) [inline .gitignore]: - [ ] You should add `test.md` file in the `.gitignore` file here. Kindly add it. :+1:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+> Description of the project.
+- In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## Built With
+
+- HTML
+- CSS
+
+### Deployment
+
```
Comment (mahbubul14) [inline README.md]: - [ ] In your `README` file, there is no content for the "Getting Started" section. Kindly add content for the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "bug fix".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ichala — APPROVED** [review]
Hi @Kasaija-Kenneth ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻
✅ You used the correct Github Flow.
✅ Professional README.
✅ No Linters errors.
✅ .gitignore looks great.


Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- [ ] [OPTIONAL] Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). kindly try to apply that in your future projects ! otherwise great job again you did a great effort



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**